include/rootston/view.h

```diff
@@ -13,6 +13,7 @@ struct roots_wl_shell_surface {
 	struct wl_listener request_move;
 	struct wl_listener request_resize;
 	struct wl_listener request_set_maximized;
+	struct wl_listener request_set_fullscreen;
```

Comment:
Naming should be consistent across the views if possible (so `request_fullscreen` and `request_mazimized`).

Reply:
I tried to respect each shell's terminology. I'll unify all of them then.